sdk/servicebus/Microsoft.Azure.ServiceBus/tests/ExpectedMessagingExceptionTests.cs

```diff
@@ -194,5 +194,19 @@ public async Task OperationsOnMessageSessionAfterCloseShouldThrowObjectDisposedE
                 }
             });
         }
+
+        [Fact]
+        [LiveTest]
+        public async Task CreatingLinkToNonExistingEntityShouldThrowEntityNotFoundException()
+        {
+            var receiver = new MessageReceiver(TestUtility.NamespaceConnectionString, "nonExistingEntity"); // Covers queue and topic
```

Comment:
@jsquire, I'm not so sure how to convert this into a unit test without mocking the behavior of link.open().
In this case, the service is returning `EntityNotFoundException` on `link.Open`.

Reply:
You're absolutely correct.  

This is my bad.   I'm conflating the changes that were made to Event Hubs with those made for Service Bus.   I had forgotten there is a static resident SB namespace present rather than it being created on the fly within the scope.  My response was to line 202 existing outside the scope... but the scenario is different than what was in my head.   Ignore this.